core/src/scss/utilities/mixins/typography/_caption.scss

```diff
@@ -8,9 +8,9 @@
 // Style guide: Mixins.Typography.caption
 //
 @mixin caption {
-  @include margin(0 null null);
-  @include modular-typography(0);
+  @include font-smoothing;
 
   color: $su-color-cool-grey;
-  font-style: italic;
+  font-size: 0.84em;
```

Comment:
Why not modular?

Reply:
Modular just a bit too small. I think the -1 step is 0.8 and it does make a difference.